GuildedRoseLLC/ItemsViewController.swift

```diff
@@ -10,14 +10,20 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         
     override public func viewDidLoad() {
         super.viewDidLoad()
-        let items = Item.getItems()
-        configureDataSource(items: items)
-        toggleListDisplay(items: items)
+        dataSource = ItemCollectionViewDataSource()
+        configureDataSource(items: [])
```

Comment:
In theory, you shouldn't need to pass any items into this method.

Reply:
Found a way to provide a default value for now since we do need to pass a list of items in the viewWillAppear fn().

Reply:
I think this ties back to the fact that there's still not a clean separation between configuring the data source and showing the items. See other comments for details. :)